html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form action="" method="get">
+      <div>
+        <label for="email">Email:</label>
+      </div>
+        <input type="text" name="email" id="email" size="15" />
+      <div>
+        Why is coding cool?
+      </div>
+      <select name="coding-cool-answer">
+        <option value="select" id="select">
+          <label for="select">Select an option</label>
```

Comment:
Individual `option` elements do not need a label, since only one will be visible at a time, the `label` should really correlate to the `select` element.